templates/workspaces/base/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-workspace-base
-version: 0.8.1
+version: 1.0.0
```

Comment:
why a major version jump?

Reply:
The main workspace blob storage was upgraded to hierarchical namespace,
because this is [one-way path](https://learn.microsoft.com/en-us/azure/storage/blobs/upgrade-to-data-lake-storage-gen2-how-to?tabs=azure-portal) and there's no way to downgrade I thought it is best to jump a major version.
@promisinganuj, is that upgrade to hierarchical namespace necessary for Azure Databricks? was there a special requirement?